lib/signs/bus.ex

```diff
@@ -868,6 +1016,36 @@ defmodule Signs.Bus do
     route ++ [{:headsign, headsign}, :no_service]
   end
 
+  defp message_audio({:headway, []}), do: []
+
+  defp message_audio(
+         {:headway,
+          [
+            %Message.Headway{
+              range: {range_low, range_high},
+              destination: destination
+            }
+          ]}
+       ) do
+    low_var = Utilities.number_var(range_low, :english)
+    high_var = Utilities.number_var(range_high, :english)
+
+    # TODO: Is there an existing message ID for "Silver Line Buses"?
+    destination_message_id =
+      case destination do
+        :chelsea ->
+          "133"
+
+        :south_station ->
+          "134"
+
+        _ ->
+          "silver_line"
```

Comment:
TBD on ARINC ID, already emailed Andrew with the new audio, just waiting to hear back.